state-chain/runtime/src/runtime_apis.rs

```diff
@@ -183,6 +184,39 @@ pub struct FailingWitnessValidators {
 	pub validators: Vec<(cf_primitives::AccountId, String, bool)>,
 }
 
+type ChainAccountFor<C> = <C as Chain>::ChainAccount;
+
+#[derive(Serialize, Deserialize, Encode, Decode, Eq, PartialEq, TypeInfo, Debug, Clone)]
+pub struct ChainAccounts {
+	pub btc_chain_accounts: Vec<ChainAccountFor<cf_chains::Bitcoin>>,
+}
+
+#[derive(Serialize, Deserialize, Encode, Decode, Eq, PartialEq, TypeInfo, Debug, Clone)]
+pub enum TaintedTransactionEvent<TxId> {
+	TaintedTransactionReportReceived {
+		account_id: <Runtime as frame_system::Config>::AccountId,
+		tx_id: TxId,
+	},
+
+	TaintedTransactionReportExpired {
+		account_id: <Runtime as frame_system::Config>::AccountId,
+		tx_id: TxId,
+	},
+
+	TaintedTransactionRejected {
+		broadcast_id: BroadcastId,
```

Comment:
nit: a comment that this is for the refund broadcast, maybe even renaming to `refund_broadcast_id`

Reply:
True, renamed.